src/index.html

```diff
@@ -855,9 +855,11 @@ <h4 class="modal-title" data-bind="locale: 'create_support_case'"></h4>
     <script src="vendors/cryptojs/rollups/aes.js"></script>
 
     <!-- bitcore -->
-    <script src="vendors/bitcore/browser/bundle.js"></script>
+    <script src="vendors/bitcore/bitcore-lib.js"></script>
```

Comment:
ehm, this might need a change before final merge ...

the `bitcore` package that is in `bower` has `bitcore/bitcore.js` but because they're renaming there's also `bitcore/bitcore-lib.js` when you do a custom build...

so as long as my repo is used this works, when the original `bitcore` package is used again this needs to be renamed!